src/util.rs

```diff
@@ -281,9 +281,16 @@ pub mod utf8 {
 	use libc::c_char;
 
 	pub fn into<'a>(data: *const c_char, length: u32) -> Vec<&'a str> {
-		unsafe {
```

Comment:
Oh I see! Sorry, I think I was the one being a bit slow yesterday.

Yeah, I think you're right. I've made a change that does this and rebased the branch. Is this what you had in mind?